src/main.rs

```diff
@@ -96,6 +96,34 @@ fn check_for_update(name: &str, version: &str) {
     }
 }
 
+fn about() -> String {
+    let mut about = String::from(
+        "Template generation tool to create no_std applications targeting Espressif's chips.\n\nThe template will use these versions:\n",
+    );
+
+    let toml = cargo::CargoToml::load(
+        TEMPLATE_FILES
+            .iter()
+            .find(|(k, _)| *k == "Cargo.toml")
+            .expect("Cargo.toml not found in template")
+            .1,
+    )
+    .expect("Failed to read Cargo.toml");
+
+    toml.visit_dependencies(|_, name, table| {
+        if name == "dependencies" {
+            for entry in table.iter() {
+                let name = entry.0;
+                if name.starts_with("esp-") {
+                    about.push_str(&format!("{} => {}\n", name, toml.dependency_version(name)));
```

Comment:
The `=>` is a bit visually noisy, and this may look nicer if the version numbers were aligned to the same column